File: Basics_with_Python/PythonBasicCourse/3_Conditional Statements Advanced/8_Conditional_Statements_Advanced - Exercise - Lab/more_exercises/01. Match Tickets.py
budged = float(input())
category = (input())
members = int(input())
transport = 0
if 1 <= members <= 4:
    transport = budged * 0.75
elif 5 <= members <= 9:
    transport = budged * 0.60
elif 10 <= members <= 24:
    transport = budged * 0.50
elif 25 <= members <= 49:
    transport = budged * 0.40
elif members >= 50:
    transport = budged * 0.25
tickets_budged = budged - transport
tickets = 0

if category == "VIP":
    tickets = 499.99 * members
    if tickets_budged >= tickets:
        print(f"Yes! You have {(tickets_budged - tickets):.2f} leva left.")
    else:
        print(f"Not enough money! You need {(tickets - tickets_budged):.2f} leva.")
elif category == "Normal":
    tickets = 249.99 * members
    if tickets_budged >= tickets:
        print(f"Yes! You have {(tickets_budged - tickets):.2f} leva left.")
    else:
        print(f"Not enough money! You need {(tickets - tickets_budged):.2f} leva.")

File: Basics_with_JavaScript/78_Conditional_Statements_Advanced/More_Exersises/01._Match_Tickets.js
function vladi(input){

    let budged = Number(input[0]);
    let category = input[1];
    let groupQty = Number(input[2])
    let transport = 0
    let tickets = 0
    let newBudged = 0
    let ticketPrice = 0
    let diff = 0

    if (groupQty >= 1 && groupQty <= 4) {
        transport = budged * 0.75;
    } else if (groupQty >= 5 && groupQty <= 9) {
        transport = budged * 0.60;
    } else if (groupQty >= 10 && groupQty <= 24) {
        transport = budged * 0.50;
    } else if (groupQty >= 25 && groupQty <= 49) {
        transport = budged * 0.40;
    } else if (groupQty >= 50) {
        transport = budged * 0.25;
        
    }

    newBudged = budged - transport;
    switch (category) {
        case "Normal":
            ticketPrice = 249.99
            break;
        case "VIP":
            ticketPrice = 499.99
            break;
    }

    tickets = ticketPrice * groupQty
    diff = (Math.abs(tickets - newBudged)).toFixed(2)

    if (newBudged >= tickets) {
        console.log(`Yes! You have ${diff} leva left.`); 
    } else {
        console.log(`Not enough money! You need ${diff} leva.`);
    }

    }
    vladi(["1000", "Normal", "1"]);
    vladi(["30000", "VIP", "49"]);